plugins/org.jkiss.dbeaver.ext.postgresql.ui/src/org/jkiss/dbeaver/ext/postgresql/tools/PostgreBackupAllWizardPageObjects.java

```diff
@@ -0,0 +1,183 @@
+import org.jkiss.utils.CommonUtils;
+
+import java.util.*;
+import java.util.List;
+
+public class PostgreBackupAllWizardPageObjects extends AbstractNativeToolWizardPage<PostgreBackupAllWizard> {
+
+    private Table databasesTable;
+
+    private PostgreDataSource dataSource;
+    private Set<PostgreDatabase> checkedObjects = new HashSet<>();
+
+    protected PostgreBackupAllWizardPageObjects(PostgreBackupAllWizard wizard) {
+        super(wizard, PostgreMessages.wizard_backup_all_page_global_backup_name);
+        setTitle(PostgreMessages.wizard_backup_page_object_title);
+        setDescription(PostgreMessages.wizard_backup_all_page_global_backup_tip);
+    }
+
+    @Override
+    public void createControl(Composite parent) {
+
+        Composite composite = UIUtils.createPlaceholder(parent, 1);
+
+        Group objectsGroup = UIUtils.createControlGroup(
+            composite,
+            PostgreMessages.wizard_backup_page_object_group_object,
+            1,
+            GridData.FILL_HORIZONTAL,
+            0);
+        objectsGroup.setLayoutData(new GridData(GridData.FILL_BOTH));
+
+        {
+            Composite catPanel = UIUtils.createComposite(objectsGroup, 1);
+            catPanel.setLayoutData(new GridData(GridData.FILL_BOTH));
+            databasesTable = new Table(catPanel, SWT.BORDER | SWT.CHECK);
+            databasesTable.addListener(SWT.Selection, event -> {
+                TableItem item = (TableItem) event.item;
+                if (event.detail == SWT.CHECK) {
+                    databasesTable.select(databasesTable.indexOf(item));
+                }
+                updateState();
+            });
+            GridData gd = new GridData(GridData.FILL_BOTH);
+            databasesTable.setLayoutData(gd);
+
+            Composite buttonsPanel = UIUtils.createComposite(catPanel, 3);
+            buttonsPanel.setLayoutData(new GridData(GridData.FILL_HORIZONTAL));
+            new Label(buttonsPanel, SWT.NONE).setLayoutData(new GridData(GridData.GRAB_HORIZONTAL));
+            createCheckButtons(buttonsPanel, databasesTable);
+        }
+
+        setControl(composite);
+    }
+
+    @Override
+    protected boolean determinePageCompletion() {
+        boolean complete = false;
+        if (!checkedObjects.isEmpty()) {
+            complete = true;
+        }
+        for (TableItem item : databasesTable.getItems()) {
+            if (item.getChecked()) {
+                complete = true;
+                break;
+            }
+        }
+        return complete && super.determinePageCompletion();
+    }
+
+    @Override
+    public void activatePage() {
+        super.activatePage();
+        loadSettings();
+        updateState();
+    }
+
+    @Override
+    public void deactivatePage() {
+        saveState();
+    }
+
+    private void loadSettings() {
+        checkedObjects.clear();
+        databasesTable.removeAll();
+
+        Set<PostgreDatabase> activeDatabases = new LinkedHashSet<>();
```

Comment:
You don't need `LinkedHashSet` because you don't retrieve its elements

```suggestion
        Set<PostgreDatabase> activeDatabases = new HashSet<>();
```